src/lib.rs.in

```diff
@@ -559,6 +559,17 @@ class DocumentDecryptResult{
     method document_decrypt_result::last_updated(&self) -> DateTime<Utc>; alias getLastUpdated;
 });
 
+foreigner_class!(
+/// Result of decrypting an unmanaged document. Includes the id and the decrypted bytes.
+class DocumentDecryptUnmanagedResult{
```

Comment:
Is there a reason you didn't provide access to the to `access_via: UserOrGroup`?

Reply:
added with `getAccessViaUserOrGroup`